instrumentation/apache-httpasyncclient-4.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apachehttpasyncclient/ApacheHttpAsyncClientTracer.java

```diff
@@ -30,6 +32,17 @@ public static ApacheHttpAsyncClientTracer tracer() {
     return TRACER;
   }
 
+  public Context startSpan(Context parentContext) {
+    Span span = super.startSpan(DEFAULT_SPAN_NAME, Span.Kind.CLIENT);
+    return parentContext.with(span);
+  }
+
+  public void inject(Context context, HttpRequest request) {
+    OpenTelemetry.getGlobalPropagators()
```

Comment:
This seems like a very boring method, do we need it?

Reply:
removed 😂